README.md

```diff
@@ -1,17 +1,27 @@
-# n5geh.tools.entirety2
+# n5geh.tools.entirety
 
-[![semantic-release](https://github.com/N5GEH/n5geh.tools.entirety2/actions/workflows/semantic-release.yml/badge.svg)](https://github.com/N5GEH/n5geh.tools.entirety2/actions/workflows/semantic-release.yml)
-[![issue-tracker](https://github.com/N5GEH/n5geh.tools.entirety2/actions/workflows/issue-tracker.yml/badge.svg)](https://github.com/N5GEH/n5geh.tools.entirety2/actions/workflows/issue-tracker.yml)
+[![semantic-release](https://github.com/N5GEH/n5geh.tools.entirety/actions/workflows/semantic-release.yml/badge.svg)](https://github.com/N5GEH/n5geh.tools.entirety/actions/workflows/semantic-release.yml)
+[![issue-tracker](https://github.com/N5GEH/n5geh.tools.entirety/actions/workflows/issue-tracker.yml/badge.svg)](https://github.com/N5GEH/n5geh.tools.entirety/actions/workflows/issue-tracker.yml)
 
 [![Conventional Commits](https://img.shields.io/badge/Conventional%20Commits-1.0.0-yellow.svg)](https://conventionalcommits.org)
 
 ## Built With
 
-- Django 4.0.5
-- Bootstrap 5.2.0-beta1
-- htmx 1.7.0
+- Django 4.1
+- Bootstrap 5.2
+- htmx 1.8.2
 
-## Getting Started
+## Deployment
+
+To get started we provide a [docker compose file](./docker-compose.yml).
```

Comment:
The docker compose files are now in the step by step guide.

Reply:
Branch was based on older version of development. Should be removed after merge.